incomfortclient/__init__.py

```diff
@@ -32,41 +32,41 @@
 # key label: displ_code
 DISPLAY_CODES: list[int, str] = {
     0: "opentherm",
-    15: "boiler ext.",
+    15: "boiler_ext",
     24: "frost",
-    37: "central heating rf",
-    51: "tapwater int.",
+    37: "central_heating_rf",
+    51: "tapwater_int",
     85: "sensortest",
-    102: "central heating",
+    102: "central_heating",
     126: "standby",
-    153: "postrun boiler",
+    153: "postrun_boiler",
     170: "service",
     204: "tapwater",
-    231: "postrun ch",
-    240: "boiler int.",
+    231: "postrun_ch",
+    240: "boiler_int",
     255: "buffer",
 }
```

Comment:
Would it maybe make sense to make this an enum?
```py
class DisplayCodes(IntEnum):
    OPENTHERM = 0
    BOILER_EXT = 15
    ....
```

Reply:
done